Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -903,6 +931,28 @@ theorem volume_preserving_pi {α' β' : ι → Type*} [∀ i, MeasureSpace (α'
     MeasurePreserving (fun (a : (i : ι) → α' i) (i : ι) ↦ (f i) (a i)) :=
   measurePreserving_pi _ _ hf
 
+/-- Docstring. -/
+theorem measurePreserving_arrowCongr' {α₁ β₁ α₂ β₂ : Type*} [Fintype α₁] [Fintype α₂]
+    [MeasurableSpace β₁] [MeasurableSpace β₂] (μ : α₁ → Measure β₁) (ν : α₂ → Measure β₂)
+    [∀ i, SigmaFinite (ν i)] (hα : α₁ ≃ α₂) (hβ : β₁ ≃ᵐ β₂)
+    (hm : ∀ i, MeasurePreserving hβ (μ i) (ν (hα i))) :
+    MeasurePreserving (MeasurableEquiv.arrowCongr' hα hβ) (Measure.pi fun i ↦ μ i)
+      (Measure.pi fun i ↦ ν i) := by
+  classical
+  convert (measurePreserving_piCongrLeft (fun i : α₂ ↦ ν i) hα).comp
+    (measurePreserving_pi μ (fun i : α₁ ↦ ν (hα i)) hm)
+  simp only [MeasurableEquiv.arrowCongr', Equiv.arrowCongr', Equiv.arrowCongr, EquivLike.coe_coe,
+    MeasurableEquiv.coe_mk, Equiv.coe_fn_mk, MeasurableEquiv.piCongrLeft, Equiv.piCongrLeft,
+    Equiv.symm_symm_apply, Equiv.piCongrLeft'_symm, Equiv.symm_symm]
+  rfl
```

Comment:
This `rfl` seems to indicate there's one or more missing lemmas

Reply:
I added some more lemmas to the `simp` and the `rfl` disappeared